drift/info_parser.py

```diff
@@ -140,6 +140,7 @@ def _group_comparisons(comparisons):
 
     # default values for method run
     grouped_comparisons = []
+    temp_grouped_comparisons_dict = {}
     processed_group_names = set()
```

Comment:
Is `processed_group_names` sill necessary?

Reply:
@fellipeh this one is still relevant

Reply:
Yes, we need to check if the group was already created, we can't move the `get` on dict (https://github.com/RedHatInsights/drift-backend/pull/348/files#diff-d94d1e0636fff7ab5a4d4f6f24ff091770a8a2ec2c170453ec644766a4f16857R171) to above it to test, since we need to get the item after the creation.. so, moving it, we need to have a second get into the dict, which is more thing to do... so.. using only on `set` to check it, is more efficient, I believe.

Reply:
Set and dict are formally the same with regards to key existence checks. The same check can be use with the dict that is being populated, cannot it?

Reply:
Fixed here: 40f4d1843e8c43cb2b7f2b2244d2c8e3d3ffc386